rootfs/etc/nginx/lua/util/split.lua

```diff
@@ -0,0 +1,59 @@
+local _M = {}
+
+-- splits strings into host and port
+function _M.parse_addr(addr)
```

Comment:
where is this being used?

Reply:
Seems like it's only here: https://github.com/Shopify/ingress/pull/53/files#diff-17c9b180fb084eb7f8281665124da6c5R38

Could probably be local if that was your point?

Reply:
oh I missed that reference

yeah in that case let's localize it